### DISCUSSION TITLE
Where has the Octal SPIRAM version gone?

### DISCUSSION THREAD

**Discussion by ma261065** [micropython/micropython#12338] [ESP32] [answered]
In https://www.micropython.org/download/ESP32_GENERIC_S3/ it mentions "However if your board has Octal SPIRAM, then use the 'spiram-oct' variant."

Where is that variant?

I don't see it under `ports/esp32/boards`

Prior to the merging of variants there used to be a `GENERIC_S3_SPIRAM_OCT` board, but it no longer exists.

**Comment by jimmo** [comment, marked as answer]
More info at https://github.com/orgs/micropython/discussions/12294

`SPIRAM_OCT` is a variant of `ports/esp32/boards/ESP32_GENERIC_S3`  (look in `mpconfigboard.cmake`)
